src/txmempool.h

```diff
@@ -356,6 +356,8 @@ enum class MemPoolRemovalReason {
     REPLACED,    //!< Removed for replacement
 };
 
+std::string MemPoolRemovalReasonString(const MemPoolRemovalReason reason);
```

Comment:
nit: const?

Reply:
Do you mean remove the const from the parameter?

Reply:
Nevermind, that's not a class method, useless to put const in function signature